### DISCUSSION TITLE
Can attributes or requirements of a particular system be generalised?

### DISCUSSION THREAD

**Discussion by JRBliekendaal** [JRBliekendaal/master-thesis#33] [Q&A] [answered]
When I use the lens of the systems theory (yes going to do this) is it possible to take a attribute or requirement or other things from eg. a software system and generalise that so it can be used on an organisational system? As I see it a system is a system but only the context changes. I see a lot of software system ideas but when you think about it it is not only applicable on a software system but also on a organisational system or a systems of systems, etc...

What I could do is state this as a lens and an approach and validate the outcome with Delphi. I see a lot of opportunities in this approach.. Maybe we should talk about this part with Hans on Thursday the 9th of September 2021.

**Comment by edzob** [comment, marked as answer]
interresting.. 
I am not certain if the logical reasoning from overlap between IT systems and organisation systems etc is easy or even possible. 

It would help in the discussion with Hans to (1) write down the assumptions in a numbered list (lens, foundational reasoning) and then (2) your conclusing research question.

The big issue here is how to scope (1) the role of the human in the discretion (red transactions) in the outcome of a "system" (2) het interrelations and effects of connected systems. 

I think I need to re-read Barrys work to determine which "foundations" he applies in his reasoning. Last time I found this the weak side of his work but he did a lot of work the last year.
